control/tests/flatsys_test.py

```diff
@@ -452,6 +452,29 @@ def test_flat_solve_ocp(self, basis):
         np.testing.assert_almost_equal(x_const, x_nlconst)
         np.testing.assert_almost_equal(u_const, u_nlconst)
 
+    def test_solve_flat_ocp_scalar_timepts(self):
+        # scalar timepts gives expected result
+        f = fs.LinearFlatSystem(ct.ss(ct.tf([1],[1,1])))
+
+        def terminal_cost(x, u):
+            return (x-5).dot(x-5)+u.dot(u)
+
+        traj1 = fs.solve_flat_ocp(f, [0, 1], x0=[23],
+                                  terminal_cost=terminal_cost)
+
+        traj2 = fs.solve_flat_ocp(f, 1, x0=[23],
+                                  terminal_cost=terminal_cost)
+
+        teval = np.linspace(0, 1, 101)
+
+        r1 = traj1.response(teval)
+        r2 = traj2.response(teval)
+
+        assert np.max(abs(r1.x-r2.x)) == 0
+        assert np.max(abs(r1.u-r2.u)) == 0
+        assert np.max(abs(r1.y-r2.y)) == 0
```

Comment:
Is this a fancy way to say `r1.x == r2.x` while employing on some opaque shape casting?

Reply:
just laziness! fixed.